pallets/lbp/src/tests.rs

```diff
@@ -1430,7 +1449,14 @@ fn remove_liquidity_from_not_started_pool_should_work() {
 
 		expect_events(vec![
 			frame_system::Event::KilledAccount(KUSD_BSX_POOL_ID).into(),
-			Event::LiquidityRemoved(KUSD_BSX_POOL_ID, KUSD, BSX, pool_balance_a_before, pool_balance_b_before).into(),
+			Event::LiquidityRemoved(
```

Comment:
Should this kind of test be a part of this pallet (maybe leftover from pre-nft)?